cmd/tap/README.md

```diff
@@ -0,0 +1,179 @@
+`tap`: atproto sync utility
+========================================
+
+Tap is a single-tenant service that subscribes to an atproto relay and outputs filtered, verified events for a subset of repos.
+
+Tap simplifies firehose consumption by handling verification, backfill, and filtering. Your application connects to tap and receives simple JSON events for only the repos and collections you care about. Historical data for configured repos is automatically fetched from PDSs and delivered before live events begin.
```

Comment:
might be worth noting that client specifically do not need to deal with binary formats (CBOR) or cryptographic signatures.